docs/development.md

```diff
@@ -186,6 +186,21 @@ To "log out", submit the following API call:
 GET http://localhost:8080/logout
 ```
 
+### How to enter masquerade mode
+Masquerade mode is a feature that enables the admin create mock instructors and students for testing and other purposes.
+- To be able to use the masquerade mode, do the following :
+    - Log in as an administrator to `http://localhost:8080/web/admin/home`.
+    - Create new instructor with by filling the Name, Email, Institution fields in the form
+      provided and submit.
+    - After adding instructor, copy the join link and paste in another browser.
+    - In the new browser, log in with the new instructor's email
+      (not the default admin email) and register to course.
+    - Now there is a new instructor with its own google id that can be used to login.
```

Comment:
Suggest removing this entirely, or changing it to "Now, we have a new account that we can masquerade as." In any case, do refrain from using "google id" as mentioned in [this comment](https://github.com/TEAMMATES/teammates/issues/12466#issuecomment-1578848527)

Reply:
I think your suggestion of rewording it is valid considering that emails for accounts are no longer restricted to google emails.